### ISSUE TITLE
1.19.3 incompatibility

### ISSUE THREAD

**Issue by SlimeDog** [danirod12/NTD-LuckyBlock#6]
Paper 1.19.3-310
ntdLuckyBlock 2.7.8

On server start-up, and every 30 seconds thereafter
```
[10:12:03 WARN]: java.lang.NoSuchMethodException: net.minecraft.world.item.ItemStack.getTag()
[10:12:03 WARN]:        at java.base/java.lang.Class.getMethod(Class.java:2277)
[10:12:03 WARN]:        at ntdLuckyBlock-2.7.8.jar//me.DenBeKKer.ntdLuckyBlock.d.e.do(ItemTagLegacy.java:268)
[10:12:03 WARN]:        at ntdLuckyBlock-2.7.8.jar//me.DenBeKKer.ntdLuckyBlock.customitem.CustomItemFactory.compare(CustomItemFactory.java:79)
[10:12:03 WARN]:        at ntdLuckyBlock-2.7.8.jar//me.DenBeKKer.ntdLuckyBlock.customitem.Identifier.compare(Identifier.java:85)
[10:12:03 WARN]:        at ntdLuckyBlock-2.7.8.jar//me.DenBeKKer.ntdLuckyBlock.customitem.luckybow.BowListener.despawn(BowListener.java:44)
[10:12:03 WARN]:        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor857.execute(Unknown Source)
[10:12:03 WARN]:        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77)
[10:12:03 WARN]:        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
[10:12:03 WARN]:        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[10:12:03 WARN]:        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:672)
[10:12:03 WARN]:        at org.bukkit.craftbukkit.v1_19_R2.event.CraftEventFactory.callItemDespawnEvent(CraftEventFactory.java:778)
[10:12:03 WARN]:        at net.minecraft.world.entity.item.EntityItem.inactiveTick(EntityItem.java:220)
[10:12:03 WARN]:        at net.minecraft.server.level.WorldServer.a(WorldServer.java:1215)
[10:12:03 WARN]:        at net.minecraft.world.level.World.a(World.java:918)
[10:12:03 WARN]:        at net.minecraft.server.level.WorldServer.lambda$tick$6(WorldServer.java:733)
[10:12:03 WARN]:        at net.minecraft.world.level.entity.EntityTickList.a(EntityTickList.java:42)
[10:12:03 WARN]:        at net.minecraft.server.level.WorldServer.a(WorldServer.java:713)
[10:12:03 WARN]:        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1535)
[10:12:03 WARN]:        at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:440)
[10:12:03 WARN]:        at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1397)
[10:12:03 WARN]:        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1173)
[10:12:03 WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316)
[10:12:03 WARN]:        at java.base/java.lang.Thread.run(Thread.java:1589)
```
Similar errors on Spigot 1.19.3-3619.

**Comment by p0t4t0sandwich** [comment]
Same issue

**Comment by danirod12** [comment]
It is clearly described on plugin page that we do not support 1.19.3 yet

**Comment by SlimeDog** [comment]
Sure. I just posted the errors I'm seeing. No worries.

**Comment by danirod12** [comment]
Added 1.19.3 compatibility with 2.7.9 update

**Comment by SlimeDog** [comment]
Confirmed fixed with 2.7.9. Thanks.
